page1.html

```diff
@@ -0,0 +1,195 @@
+
```

Comment:
I wonder if we can't get both page1 and page2 to coexist on a single page. Where clicking a button would hide the one view and show the other?